mycore-ocfl/src/main/java/org/mycore/ocfl/user/MCROCFLUserEventHandler.java

```diff
@@ -33,28 +34,30 @@ public class MCROCFLUserEventHandler implements MCREventHandler {
 
     private static final Logger LOGGER = LogManager.getLogger();
 
-    private static final MCROCFLXMLUserManager MANAGER = new MCROCFLXMLUserManager();
+    private static final MCROCFLXMLUserManager MANAGER
+        = MCRConfiguration2.<MCROCFLXMLUserManager>getSingleInstanceOf("MCR.OCFL.User.Manager")
+            .orElseThrow();
```

Comment:
As stated with Classifications, there is no need to allow non-ocfl managers in an event handler specifically designed for the ocfl xml manager.